frontend/src/data/nsf313_20/categories.ts

```diff
@@ -0,0 +1,18 @@
+export const Categories = {
+    'race': new Set<string>([
+        'Black'
+    ]),
+    'collegeOrUniversityLevel': new Set<string>([
+        '4year',
+        '2year',
+    ]),
+    'collegeOrGraduateSchoolEnrollment': new Set<string>([
+        'EnrolledInCollegeOrGraduateSchool', 
```

Comment:
Since `EnrolledInCollegeOrGraduateSchool` is a summary statistic for the 2 dimensions below, can we simply ignore it?

Reply:
Yup we can, how does that translate for filters? Are we consistently removing summary statistics on all sets?

Reply:
A query for all dimensions is the same as omitting all dimensions- we use the `${category}:` key as a shortcut for all dimensions. If there's a dimension in a category that implies all dimensions, so far it's made sense to remove it and retain the "all means none" dimension query logic.

Reply:
Additionally, if there's a summary dimension that is the sum of a subset of all dimensions in a category, it's so far made sense to remove that as well. We'll be able to sum multiple dimensions within a category using the plotting logic.

Reply:
Gotcha, summary statistic omitted and extra query logic removed